api/usePlan.js

```diff
@@ -1,5 +1,5 @@
 import { useState } from 'react';
-import useSWR from 'swr';
+import useSWR, { mutate as globalMutate } from 'swr';
```

Comment:
I don't think we need this any more, didn't end up being required.

```suggestion
import useSWR from 'swr';
```